src/Microsoft.PowerShell.Commands.Utility/commands/utility/CSVCommands.cs

```diff
@@ -206,8 +206,9 @@ public SwitchParameter NoClobber
         /// Encoding optional flag
         /// </summary>
         [Parameter()]
-        [ValidateSetAttribute(new string[] { "Unicode", "UTF7", "UTF8", "ASCII", "UTF32", "BigEndianUnicode", "Default", "OEM" })]
-        public string Encoding { get; set; }
+        [ArgumentToEncodingTransformationAttribute()]
```

Comment:
I think we need to add `AttributeNotNullOrEmpty` attribute. Before this change, $null value would be rejected because the validate attribute will fail. After this change, user is able to pass in a null value.